packages/iov-keycontrol/src/hdpaths.ts

```diff
@@ -0,0 +1,13 @@
+import { Slip10RawIndex } from "@iov/crypto";
+
+export class HdPaths {
+  /**
+   * IOV's SimpleAddress derivation path
+   *
+   * @see https://github.com/iov-one/iov-core/blob/v0.6.1/docs/KeyBase.md#simple-addresses
+   */
+  public static simpleAddress(index: number): ReadonlyArray<Slip10RawIndex> {
```

Comment:
Okay, nice. So this is the first of various path derivations.

Later bip44, etc... or use raw for full bip32

Reply:
It would be nice to see a test (later once implemented) with different path derivations in the same keyring.

one simpleAddress, one bip44, one bip44 with another chain, etc.